### PR TITLE
[SC-705] Chore: correct spelling

### PR DESCRIPTION
**Pull request by Zitzak**

### CODE REVIEW

File: src/modules/authorizer/role/AUT_Roles_v1.sol [hunk 1]
```diff
@@ -131,13 +131,10 @@ contract AUT_Roles_v1 is
         _setRoleAdmin(DEFAULT_ADMIN_ROLE, ORCHESTRATOR_OWNER_ROLE);
 
         // Set up MANAGER role structure:
```
Comment (marvinkruse) [inline src/modules/authorizer/role/AUT_Roles_v1.sol]: `_setRoleAdmin` doesn't have a specific line documenting its behavior anymore, as the like `Set up MANAGER... ` is the overall heading of what's happening in all of these lines. Was this intentional? I'd have left that line in.

File: src/modules/authorizer/role/AUT_Roles_v1.sol [hunk 2]
```diff
@@ -131,13 +131,10 @@ contract AUT_Roles_v1 is
...
-        // -> set OWNER as admin of DEFAULT_ADMIN_ROLE
         _setRoleAdmin(ORCHESTRATOR_MANAGER_ROLE, ORCHESTRATOR_OWNER_ROLE);
         // grant MANAGER Role to specified address
         _grantRole(ORCHESTRATOR_MANAGER_ROLE, initialManager);
-
-        // If there is no initial owner specfied or the initial owner is the same as the deployer
-
+        // grant Owner Role to initial owner address
```
Comment (marvinkruse) [inline src/modules/authorizer/role/AUT_Roles_v1.sol]: Above, the role names are capitalized (`MANAGER`, `OWNER`), here the first `Owner` is not

**Review by marvinkruse — CHANGES_REQUESTED** [review]
Otherwise good :)

File: src/modules/authorizer/role/AUT_Roles_v1.sol (hunk 1 above)
Comment (Zitzak) [reply src/modules/authorizer/role/AUT_Roles_v1.sol]: Followed Omega's advice. However see your point in this

**Review by marvinkruse — APPROVED** [review]
thank you, lgtm!

**Pull request merged**